packages/site/cms-site/src/blocks/PixelImageBlock.tsx

```diff
@@ -111,3 +96,9 @@ function createDominantImageDataUrl(w: number, h: number, dominantColor = "#ffff
 
     return `data:image/svg+xml;base64,${toBase64(svg)}`;
 }
+
+const ImageContainer = styled.div<{ $aspectRatio: number }>`
+    position: relative;
+    width: 100%;
+    aspect-ratio: ${({ $aspectRatio }) => $aspectRatio};
+`;
```

Comment:
Why is this needed? I can imagine that this leads to many unwanted behaviors

Reply:
as discussed, the container is needed for our default behaviour